setup.sh

```diff
@@ -42,11 +42,21 @@ function _unset_vars
   unset SCRIPT USING_SELINUX
 }
 
-function _get_current_directory
+function _get_script_path
 {
-  if dirname "$(readlink -f "${0}")" &>/dev/null
+  if uname -a | grep -q Darwin
```

Comment:
``` DIFF
- if uname -a | grep -q Darwin
+ grep -q "Darwin" < <(uname -a)
```

Reply:
Personal taste. I find my variant more readable in that case.

Reply:
The good, old piping into `grep` :D Alright, fine with me :)